pkg/v1/tarball/write.go

```diff
@@ -55,9 +55,15 @@ func MultiWriteToFile(p string, tagToImage map[name.Tag]v1.Image, opts ...WriteO
 // MultiRefWriteToFile writes in the compressed format to a tarball, on disk.
 // This is just syntactic sugar wrapping tarball.MultiRefWrite with a new file.
 func MultiRefWriteToFile(p string, refToImage map[name.Reference]v1.Image, opts ...WriteOption) error {
-	w, err := os.Create(p)
-	if err != nil {
-		return err
+	var w io.WriteCloser
+	var err error
+	if p == "-" {
+		w = os.Stdout
+	} else {
+		w, err = os.Create(p)
+		if err != nil {
+			return err
+		}
```

Comment:
Move this logic into `crane.MultiSave` and have it call `tarball.MultiRefWrite` directly instead of this method.

Reply:
Called `tarball.MultiWrite` instead of `tarball.MultiRefWrite`, because `crane.MultiSave` construct `tagToImage` instead of `refToImage`.

Reply:
Replaced `tarball.MultiWrite` in favor of `tarball.MultiRefWrite` after cleaning out the old code.